src/wp-includes/class-wp-image-editor-imagick.php

```diff
@@ -745,6 +749,27 @@ public function save( $destfilename = null, $mime_type = null ) {
 		return $saved;
 	}
 
+	/**
+	 * Remove PDF transparency after it's been read.
```

Comment:
```suggestion
	 * Removes PDF transparency after it's been read.
```
Using a third-person singular verb to start the docblock.